x-pack/plugin/src/test/resources/rest-api-spec/test/data_stream/130_migrate_to_data_stream.yml

```diff
@@ -0,0 +1,69 @@
+---
+"Migrate to data stream":
+  - skip:
+      version: " - 7.99.99"
+      reason: "change to 7.10.99 after backport"
```

Comment:
I think `change to 7.10.99 after backport` should be `change to 7.9.99 after backport`?

Reply:
I don't think the migrate feature will be available until the 7.11 release?

Reply:
🤦 true, I somehow always think the feature makes it into the 7.10 release :)